ml-notebook/packages.txt

```diff
@@ -215,9 +214,7 @@ gcc_impl_linux-64==13.3.0
 gcc_linux-64==13.3.0
 gcm_filters==0.3.0
 gcsfs==2024.9.0
-gdal==3.9.2
```

Comment:
Interesting, gdal command line tools it turns out have been coming from pyogrio, with the latest release it's just  `libgdal-core` dependency https://github.com/geopandas/pyogrio/releases/tag/v0.10.0

Reply:
Not just `pyogrio` but also `rasterio` has switched to `libgdal-core`, see https://github.com/conda-forge/pyogrio-feedstock/issues/48 and https://github.com/conda-forge/rasterio-feedstock/issues/303.